### PR TITLE
Update languages

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

**Comment by BigRoma** [comment]
@kkkhesha Please, add Korean translation.

**Review by sertony — APPROVED** [review]

File: src/ko.json [hunk 1]
```diff
@@ -176,6 +176,9 @@
             "info": {
                 "about": "자세한 내용",
                 "actions": "기록",
+                "legalNotes": "Legal notes",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "legalNotes": "법적 고지",
```

File: src/ko.json [hunk 2]
```diff
@@ -176,6 +176,9 @@
...
+                "sourceCode": "Source code",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "sourceCode": "소스 코드",
```

File: src/ko.json [hunk 3]
```diff
@@ -176,6 +176,9 @@
...
+                "share": "Share",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "share": "공유",
```

File: src/ru.json [hunk 4]
```diff
@@ -172,6 +172,9 @@
             "info": {
                 "about": "Информация",
                 "actions": "Действия",
+                "legalNotes": "Legal notes",
```
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
                "legalNotes": "Правовая информация",
```

**Review by sertony — APPROVED** [review]

**Pull request merged**